README.md

```diff
@@ -24,21 +24,38 @@ To run that demo on your own computer:
 * `npm run storybook`
 * Visit http://localhost:9001/
 
-## How to Use
+## Getting Started
+#### Install dependencies
+```
+npm install --save react-dates
+```
+Make sure you have the following dependencies installed:
+```
+ npm install --save moment react react-dom react-addons-shallow-compare
```

Comment:
@ljharb alright I'm good with this now that I see it runs in Git Bash... I couldn't get it to run under Cygwin or Cmder which is why I put the script together in the first place.. will put in an updated PR soon